apps/design/backend/src/app.test.ts

```diff
@@ -885,7 +895,7 @@ test.skip('Election package export', async () => {
   //
   // Check uiStringAudioIds.json
   //
-  expect(countObjectLeaves(uiStringAudioIds)).toEqual(
+  expect(countObjectLeaves(uiStringAudioIds)).toBeGreaterThanOrEqual(
```

Comment:
Still figuring out why this went up

Reply:
[e1beae4](https://github.com/votingworks/vxsuite/pull/6088/commits/e1beae4e2f9b89bbd9c72444022bf811d69c15fd)